### ISSUE TITLE
Error at postgresql-pgaudit-9.6 files

### ISSUE THREAD

**Issue by sfbenitez** [dhatim/pgaudit-deb#6]
Hi dhatim, i'm using your deb pkg for postgresql 9.6 and i see an error on the pkg files.

After install the pkg, i receive the next log msg:
```
2018-02-04 22:48:57.881 UTC [1397] FATAL:  can't access to the file «pgaudit»: No such file or directory
```
Then, loocking the pkg files, i saw something wrong.
```
root@pandora:~# dpkg -L postgresql-pgaudit-9.6
/usr
/usr/lib
/usr/lib/postgresql
/usr/lib/postgresql/10
/usr/lib/postgresql/10/lib
/usr/lib/postgresql/10/lib/pgaudit.so
/usr/share
/usr/share/postgresql
/usr/share/postgresql/9.6
/usr/share/postgresql/9.6/extension
/usr/share/postgresql/9.6/extension/pgaudit.control
/usr/share/postgresql/9.6/extension/pgaudit--1.1.1.sql
```
After move the file pgaudit.so to /usr/share/postgresql/9.6/lib, the extension seems working, but i don know if that lib is the correct version for PostgreSQL 9.6.

If you think there is no error, just close this issue.
pd:sorry for my english...spanish guy here, still learning.

**Comment by ochedru** [comment]
Oops, you are right: there was a typo in 9.6's `pom.xml`. Fixed with PR https://github.com/dhatim/pgaudit-deb/pull/7. A new release will follow.
Thank you for reporting this issue!

**Comment by ochedru** [comment]
Fixed in `postgresql-pgaudit-9.6` version 1.1.1.44.
